persistence/remote/src/main/java/org/infinispan/persistence/remote/configuration/RemoteStoreConfiguration.java

```diff
@@ -58,16 +65,16 @@ public static AttributeSet attributeDefinitionSet() {
    private final Attribute<String> marshaller;
    private final Attribute<ProtocolVersion> protocolVersion;
    private final Attribute<String> remoteCacheName;
-   private final Attribute<List<RemoteServerConfiguration>> servers;
    private final Attribute<Long> socketTimeout;
    private final Attribute<Boolean> tcpNoDelay;
    private final ConnectionPoolConfiguration connectionPool;
    private final ExecutorFactoryConfiguration asyncExecutorFactory;
    private final SecurityConfiguration security;
+   private List<RemoteServerConfiguration> servers;
 
    public RemoteStoreConfiguration(AttributeSet attributes, AsyncStoreConfiguration async,
                                    ExecutorFactoryConfiguration asyncExecutorFactory, ConnectionPoolConfiguration connectionPool,
-                                   SecurityConfiguration security) {
+                                   SecurityConfiguration security, List<RemoteServerConfiguration> servers) {
```

Comment:
So we will have access to other Infinispan Clusters from the console ?

Reply:
If you get the server config from REST, those servers will be listed, but they are the servers configured in the remote store for a cache

Reply:
and those server are going to be shutdown after the rolling upgrade, not sure how useful it is